### PR TITLE
hey i changed this

### PR DESCRIPTION
**Pull request by mbfrank2**
i added a note in ED cognitions

### CODE REVIEW

File: 04-ed-cognitions.Rmd [hunk 1]
```diff
@@ -5,7 +5,7 @@ Eating disorder cognitions were assessed in-depth at age 14 years, with some add
 ```{r}
 #insert table here with constructs, ages, and checkmarks
 ```
-
```
Comment (kschaumb) [inline 04-ed-cognitions.Rmd:8]: I don't think you should have deleted this

File: 04-ed-cognitions.Rmd [hunk 2]
```diff
@@ -5,7 +5,7 @@ Eating disorder cognitions were assessed in-depth at age 14 years, with some add
...
+MAX changed this
```
Comment (kschaumb) [inline 04-ed-cognitions.Rmd:8]: this won't work with filepathing

File: 04-ed-cognitions.Rmd (hunk 1 above)
Comment (mbfrank2) [reply 04-ed-cognitions.Rmd:8]: i think its fine

Comment (kschaumb) [reply 04-ed-cognitions.Rmd:8]: ok

**Review by kschaumb — APPROVED** [review]
approved for pR

**Pull request merged**